ArduPlane/systemid.cpp

```diff
@@ -155,15 +153,17 @@ void AP_SystemID::stop()
         attitude_offset_deg.zero();
         throttle_offset = 0;
 
-        auto *attitude_control = plane.quadplane.attitude_control;
-        attitude_control->bf_feedforward(restore.att_bf_feedforward);
-        attitude_control->rate_bf_roll_sysid(0);
-        attitude_control->rate_bf_pitch_sysid(0);
-        attitude_control->rate_bf_yaw_sysid(0);
-        plane.quadplane.pos_control->set_xy_control_scale_factor(1);
+        if (plane.quadplane.enabled()) {
```

Comment:
we'll need some ifdef magic if we want users to be able to run sysid on builds with quadplane not compiled in. Not sure it is worth it?

Reply:
Done.